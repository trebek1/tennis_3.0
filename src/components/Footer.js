// @flow strict-local

import React from 'react';

const Footer = () => (
  <div className="footer">
    <a
      className="alex noDecoration link"
      href="https://www.alextrz.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      About the author
    </a>
  </div>
);

export default Footer;
