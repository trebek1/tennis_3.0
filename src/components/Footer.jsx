import React from "react";

const Footer = () => (
  <div className="footer">
    <a
      className="alex noDecoration link"
      rel="noopener noreferrer"
      href="https://www.alextrz.com"
      target="_blank"
    >
      About the author
    </a>
  </div>
);

export default Footer;
