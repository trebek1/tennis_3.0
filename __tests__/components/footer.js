import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Footer from "../../src/components/footer";

describe(`Footer component`, () => {
  test("renders the footer", () => {
    const component = shallow(<Footer />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
