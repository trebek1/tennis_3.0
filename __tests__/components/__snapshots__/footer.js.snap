// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the footer 1`] = `
<div
  className="footer"
>
  <a
    className="alex noDecoration link"
    href="https://www.alextrz.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    About the author
  </a>
</div>
`;
